### DISCUSSION TITLE
GitHub Student Benefits Not Activated Despite Approval

### DISCUSSION THREAD

**Discussion by marcus6n** [community/community#136520] [GitHub Education]
### Hi everyone,

Dear GitHub Support Team,

I hope this message finds you well. I am writing to bring to your attention an issue I am facing with the activation of my GitHub Student benefits. Although my application for the GitHub Student Developer Pack has been approved, I have noticed that the benefits have not yet been activated on my account.

`Congrats! Your request for @marcus6n  was approved and your benefits will be available soon`

I have waited for the specified processing time, but the benefits still do not appear in my account. Could you please assist me in resolving this issue? I would greatly appreciate your prompt attention to this matter.

Thank you for your time and support.

Best regards,

Marcus Vinícius de Oliveira Neves

**Comment by davevad93** [comment]
Hi @marcus6n ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @marcus6n was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
